src/app/components/dashboard/shared/dashboard-header/dashboard-header.component.ts

```diff
@@ -8,9 +8,9 @@ import { LogoutComponent } from '../logout/logout.component';
 })
 export class DashboardHeaderComponent implements OnInit {
   navigation = ['Candidates', 'Planning', 'Educational programs', 'Letters', 'Report'];
-
+  user = localStorage.getItem('email');
```

Comment:
Is this a good idea to have email in local storage? what if user logouts? are you deleting it? also what if session will timeout?